.vortex/CLAUDE.md

```
@@ -25,55 +25,173 @@ vortex/
+- **MDX** - Markdown with React components
+- **Jest** - Unit testing framework
+- **ESLint/Prettier** - Code quality tools
+- **cspell** - American English spellcheck validation
 
-- Unit and functional tests for Vortex
-- Uses both **PHPUnit** (functional workflows) and **BATS** (shell script unit tests)
+**Key Features**:
 
-## Testing Framework
+- Interactive documentation with custom React components
+- Published to https://www.vortextemplate.com
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)